ui/src/components/stepforms/JoinStepForm.vue

```diff
@@ -131,11 +131,15 @@ export default class JoinStepForm extends BaseStepForm<JoinStep> {
   }
 
   get options(): object[] {
-    return this.availableDomains.map((d) => ({
-      label: d.name,
-      trackBy: { type: 'ref', uid: d.uid },
-      $isDisabled: !!this.unjoinableDomains.find((domain) => domain.uid === d.uid),
-    }));
+    return this.availableDomains.map((d) => {
+      const isDisabled = !!this.unjoinableDomains.find((domain) => domain.uid === d.uid);
+      return {
+        label: d.name,
+        trackBy: { type: 'ref', uid: d.uid },
+        disabled: isDisabled,
+        tooltip: isDisabled ? 'This dataset cannot be combined with the actual one' : '',
```

Comment:
Si je dis pas de betise je crois que tu peux utiliser cette syntaxe pour ne pas afficher l'attribut tooltip dans le cas où il est vide
```suggestion
...(isDisabled) && {tooltip: 'This dataset cannot be combined with the actual one'},
```

Reply:
Bonne idée! Sinon juste mettre undefined est OK aussi